frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -315,6 +315,28 @@ class ManageAccounts extends Component<Props, State> {
                           onInput={e => this.setState({ currentlyEditedAccount: { ...currentlyEditedAccount, name: e.target.value } })}
                           value={currentlyEditedAccount.name}
                         />
+                        <Label
+                          className={style.toggleLabel}
+                          htmlFor={currentlyEditedAccount.code}>
+                          <span className={style.toggleLabelText}>
+                            <EyeOpenedDark />
+                            {t('newSettings.appearance.enableWallet.title')}
+                          </span>
+                          <Toggle
+                            checked={currentlyEditedAccount.active}
+                            className={style.toggle}
+                            id={currentlyEditedAccount.code}
+                            onChange={(event) => {
+                              event.target.disabled = true;
+                              this.setState({
+                                currentlyEditedAccount: {
+                                  ...currentlyEditedAccount,
+                                  active: event.target.checked,
+                                }
```

Comment:
similar to how the name is temporary stored in currentlyEditedAccount, we can do the same here until the dialog is either closed or the user clicked the update button in the dialog.